go/api/command_options.go

```diff
@@ -41,6 +42,30 @@ func (opts *SetOptions) toArgs() []string {
 	return args
 }
 
+// GetExOptions represents optional arguments for the [api.StringCommands.GetExWithOptions] command.
+//
+// See [valkey.io]
+//
+// [valkey.io]: https://valkey.io/commands/getex/
+type GetExOptions struct {
+	// If not set, no expiry time will be set for the value.
+	// Supported ExpiryTypes ("EX", "PX", "EXAT", "PXAT", "PERSIST")
+	Expiry *Expiry
+}
+
+func (opts *GetExOptions) getExOptionsToArgs() []string {
```

Comment:
Suggest having toArgs() only for better naming.

Reply:
+1
You'll have a lot of option classes, all of them should have `toArgs()` method.

Reply:
Done